metashare/api/jobs.py

```diff
@@ -470,3 +472,68 @@ def populate_github_users(repository):
 
 
 populate_github_users_job = job(populate_github_users)
+
+
+def submit_review(*, user, task, data):
+    try:
+        review_sha = None
+        org = data["org"]
+        task.refresh_from_db()
+        if org:
+            review_sha = org.latest_commit
+        elif task.review_valid:
+            review_sha = task.review_sha
+
+        if task.commits and task.pr_is_open and review_sha:
```

Comment:
@wlonk I realized this isn't quite right. In the case where this `if` fails, it's still calling `finalize_submit_review` as if a review was created. Instead we should probably `raise` an Error here?

Reply:
Fixed in https://github.com/oddbird/MetaShare/pull/164/commits/4773ab7ba48054525a4e8d59f0e7a8ea33c903f8